src/utils.ts

```diff
@@ -103,7 +103,8 @@ export function jitiInteropDefault(ctx: Context, mod: any) {
 }
 
 function interopDefault(mod: any): any {
-  if (!mod || !("default" in mod)) {
+  const modType = typeof mod;
+  if (mod === null || (modType !== "object" && modType !== "function")) {
```

Comment:
What if mod is `undefined`? I'd add this check as well

Reply:
`typeof undefined` is `undefined` (which passes the second part of the logic). only `null` is odd because `typeof null === 'object'`

Reply:
ACK